lib/android/README.md

```diff
@@ -17,10 +17,30 @@ The following are also required for the compilation of the vendored openssl:
 - Perl; and
 - Clang (to be installed with the NDK, which should be included with CMake when if you use Android Studio).
 
-## Build
+## Build and integrate
+
+Build the `.so` libraries for each support ABI.
+
+```sh
+$ make -C ../ ../target/test/android.stamp
+```
+
```

Comment:
I had to set the SDK root path otherwise I got an error. Should this be mentioned here?
```suggestion
Set environmental variables as needed for Gradle:
​```
$ export ANDROID_SDK_ROOT=~/Android/Sdk
​```
```

Reply:
The variables needed are listed on the end of the readme, maybe put them on top and be more explicit on what's is needed.

Going to improve the docs, thanks!

Reply:
The other two variables, `ANDROID_NDK_HOME` and `ANDROID_TOOLS`, don't seem to have an affect on Gradle. `ANDROID_SDK_ROOT` works for both Gradle and the Makefile.

The defaults mentioned for the Make variables don't apply to environmental variables outside the Makefile. But the other way works - the Make variables can be set via environmental variables.